skunkworks_crow/src/test/java/org/odk/share/activities/HpReceiverActivityTest.java

```diff
@@ -0,0 +1,52 @@
+package org.odk.share.activities;
+
+
+import androidx.appcompat.widget.Toolbar;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.share.R;
+import org.odk.share.views.ui.hotspot.HpReceiverActivity;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.LooperMode;
+import org.robolectric.shadows.ShadowLog;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.robolectric.annotation.LooperMode.Mode.PAUSED;
+
+@RunWith(RobolectricTestRunner.class)
+@LooperMode(PAUSED)
+public class HpReceiverActivityTest {
```

Comment:
It would be awesome if we can add some cases to verify the form transfer, not necessary in this PR, we can do that in different but we should do that as well, we may have to take help of Mockito in that case to mock the sender's behaviour.